ruma-federation-api/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Improvements:
   openid::get_openid_userinfo::v1,
   query::get_profile_information::v1,
   transactions::send_transaction_message::v1,
+  user::get_devices::v1,
```

Comment:
This is the changelog section for 0.0.3, which has already been released. Please add a new `Improvements` section with an entry under `[unreleased]` instead.